lib/modules/manager/gomod/update.spec.ts

```diff
@@ -318,6 +318,26 @@ describe('modules/manager/gomod/update', () => {
       expect(res).toContain('github.com/caarlos0/env/v6 v6.1.0');
     });
 
+    it('handles multiline replace update', () => {
+      const fileContent = `
+      go 1.18
+      replace (
+        k8s.io/client-go => k8s.io/client-go v0.21.9
```

Comment:
add 1 more line to the test data

Reply:
`updateDependency` takes only one upgrade at a time. Seems redundant to create another test for a single replace since  there is already one